lib/vsc/mympirun/rm/sched.py

```diff
@@ -93,7 +93,7 @@ class Sched(object):
     RSH_LARGE_LIMIT = 16
 
     HYDRA_RMK = []
-    HYDRA_LAUNCHER = ['ssh']
+    HYDRA_LAUNCHER = ['pbsdsh']
```

Comment:
I would include a version check, and use `pbsdsh` by default for `Intel MPI` v5.x and up, and stick to `ssh` else (e.g. also for other MPI libraries like Open MPI, for now)

That probably means overriding `HYDRA_LAUNCHER` in `intelmpi.py` to `pbsdsh` if v5.x or newer, and leaving the default to be `ssh` like it is now?
That, and make sure you're not using `pbsssh`?